examples/todos/common/src/commonMain/kotlin/org/reduxkotlin/examples/todos/AppState.kt

```diff
@@ -0,0 +1,35 @@
+package org.reduxkotlin.examples.todos
+
+
+/**
+ * Entire state tree for the app.
+ */
+data class AppState(
+    val todos: List<Todo> = listOf(),
+    val visibilityFilter: VisibilityFilter = VisibilityFilter.SHOW_ALL
+) {
+    val visibleTodos: List<Todo>
+        get() = getVisibleTodos(visibilityFilter)
+
+    private fun getVisibleTodos(visibilityFilter: VisibilityFilter) = when (visibilityFilter) {
+        VisibilityFilter.SHOW_ALL -> todos
+        VisibilityFilter.SHOW_ACTIVE -> todos.filter { !it.completed }
+        VisibilityFilter.SHOW_COMPLETED -> todos.filter { it.completed }
+    }
+}
+
+/**
+ * A model of data in out state.  Where these types of models are located is up to you and your team
+ * It will likely make sense to place separate file in a real project.
+ */
+data class Todo(val text: String,
```

Comment:
I believe the Kotlin style guide would have formatted this like: 

```kotlin
data class Todo(
    val text: String,
    val completed: Boolean = false
)
```

Reply:
done